nums = [1, 2, 3]
try:
    print(nums[3])
except Exception as e:
    print(e)

print("Hello")